numpy/lib/stride_tricks.py

```diff
@@ -179,7 +179,7 @@ def _broadcast_shape(*args):
     supplied arrays against each other.
     """
     if not args:
-        raise ValueError('must provide at least one argument')
+        return ()
     # use the old-iterator because np.nditer does not handle size 0 arrays
     # consistently
     b = np.broadcast(*args[:32])
```

Comment:
I slightly misunderstood -- I agreed with your change here for sure. I'm less sure about messing with the C code, but this is mostly because quite a while ago, on earlier changes in `stride_tricks`, there was the following https://github.com/numpy/numpy/pull/5371#issuecomment-67233157